content/docs/5.1/deploy/benchmark/benchmark-instructions.md

```diff
@@ -0,0 +1,189 @@
+
+
+{{< info >}}
+You can install [TiUP](https://github.com/pingcap/tiup) as described in [TiKV in 5 Minutes](../../tikv-in-5-minutes).
+{{< /info >}}
+
+## Step 2. Deploy a cluster
+
+1. We could use the following topology to deploy our benchmark cluster via `tiup cluster`. **For more information, see [Production Deployment](../../install/production)**. Save the content below as `topology.yaml`:
+```yaml
+global:
+  user: "tikv"
+  ssh_port: 22
+  deploy_dir: "/tikv-deploy"
+  data_dir: "/tikv-data"
+server_configs: {}
+pd_servers:
+  - host: 10.0.1.1
+tikv_servers:
+  - host: 10.0.1.2
+  - host: 10.0.1.3
+  - host: 10.0.1.4
+monitoring_servers:
+  - host: 10.0.1.5
+grafana_servers:
+  - host: 10.0.1.6
+```
+
+2. Deploy the cluster:
+```sh
+tiup cluster deploy [cluster-name] [cluster-version] topology.yaml
+```
+
+3. Start the cluster:
+```sh
+tiup cluster start [cluster-name]
+```
+
+4. You can inspect the cluster information:
+```sh
+tiup cluster display [cluster-name]
+```
+
+## Step 3. Run a YCSB workload
+
+YCSB has six kinds of workloads whose main difference are the portion of different operations:
+
+1. Workload A: Update heavy workload
+2. Workload B: Read mostly workload
+3. Workload C: Read only
+4. Workload D: Read latest workload
+    > In this workload, new records are inserted, and the most recently inserted records are the most popular. Application example: user status updates; people want to read the latest.
+5. Workload E: Short ranges
+    > In this workload, short ranges of records are queried, instead of individual records. Application example: threaded conversations, where each scan is for the posts in a given thread (assumed to be clustered by thread id).
+6. Workload F: Read-modify-write 
+    > In this workload, the client will read a record, modify it, and write back the changes.
+
+All six workloads have a data set which is **similar**. Workloads D and E insert records during the test run. Thus, to keep the database size consistent, we recommend the following sequence:
+
+1. Load the database, using workload A's parameter file (workloads/workloada) .
+
+    ```sh
+    go-ycsb load -P workloads/workloada -p ... 
+    ```
+
+2. Run workload A (using workloads/workloada) for a variety of throughputs.
+
+    ```sh
+    go-ycsb run -P workloads/workloada -p ... 
+    ```
+
+3. Run workload B (using workloads/workloadb) for a variety of throughputs.
+4. Run workload C (using workloads/workloadc) for a variety of throughputs.
+5. Run workload F (using workloads/workloadf) for a variety of throughputs.
+6. Run workload D (using workloads/workloadd) for a variety of throughputs. This workload inserts records, increasing the size of the database.
+7. Delete the data in the database.
+   > You need to destroy the cluster via `tiup cluster destroy [cluster-name]`, and delete the data directory of cluster. Otherwise, the remaining data of the cluster may impact the results of the following workload.
```

Comment:
May better to wrap the command in a code block as L187.

Reply:
It will be strange in blockquotes